replication_parameters.go

```diff
@@ -47,6 +47,12 @@ type ReplicationParameters struct {
 	Async            bool
 }
 
+func NewReplicationParameters() *ReplicationParameters {
```

Comment:
Since this is only used for testing, should it get moved into the test class (or just have the test initialize it as below?)

It feels like there's some risk of non-test code using this in future, and inadvertently overriding the per-server ChangesFeedLimit.